Lib/socket.py

```diff
@@ -463,6 +465,36 @@ def fromfd(fd, family, type, proto=0):
     nfd = dup(fd)
     return socket(family, type, proto, nfd)
 
+_GLOBAL_DEFAULT_MAXFDS = object()
+
+if hasattr(_socket.socket, "sendmsg"):
+    def send_fds(sock, buffers=None, fds=[], flags=[], address=None):
+        """ send_fds(sock, buffers, fds[, flags[, address]]) -> socket object
+
+        Send the list of file descriptors fds over an AF_UNIX socket.
+        """
+        return sock.sendmsg([buffers], [(_socket.SOL_SOCKET,
+            _socket.SCM_RIGHTS, array.array("i", fds))])
+    __all__.append("send_fds")
+
+if hasattr(_socket.socket, "recvmsg"):
+    def recv_fds(sock, bufsize=0, maxfds=_GLOBAL_DEFAULT_MAXFDS, flags=0):
```

Comment:
Like ``recvmsg`` make ``bufsize`` a mandatory argument and also require the user to pass down a sensible maxfds argument:

``recv_fds(sock, bufsize, maxfds, flags=0)``